scripts/xe-reset-networking

```diff
@@ -85,12 +85,22 @@ if __name__ == "__main__":
 		try:
 			conf = read_management_conf()
 			device = conf['LABEL']
+                        # Get the VLAN if provided in the firstboot data and not specified by the user
+                        if options.vlan == None:
```

Comment:
If VLAN block of code is moved outside similar to as device is there.
There will be one use case issue: `xe-reset-networking --device=eth1` can be asked intentionally to set management on eth1. But utility will start picking vlan from firstboot as vlan is not mentioned. Output can be `device: eth1 and vlan: xxx`. Which seems to be wrong.